frontend/PreferencesPanel.js

```diff
@@ -90,7 +90,7 @@ class PreferencesPanel extends React.Component {
               ref={this._setSelectRef}
               value={themeName}
             >
-              {!showHiddenThemes && (<option value="">{browserName}</option>)}
+              {browserName && (<option value="">{browserName}</option>)}
```

Comment:
This change would make the UX a bit weird.

The themes menu would have both "_Chrome_" and "_Chrome (default)_" and "_Chrome (dark)_" options, and selecting "_Chrome_" would actually select one of the other 2. That's confusing.

I wonder if `setBrowserName` should also toggle `showHiddenThemes` to false? That would also be a bit weird I guess, if the user had pre-selected one of the "hidden" themes (eg maybe the user _prefers_ to run with a Firefox theme even though "Chrome" has been injected as the `browserName`.)

To be honest I'm still a little unclear on the benefit of this feature (matching theme between standalone devtools and a browser).